playbooks/responses/community-discord.md

```diff
@@ -0,0 +1,3 @@
+Thanks for reaching out and logging this issue! Because we treat our issues list as the team's backlog, we close issues that are questions since they don't represent a task needing to be completed. For most questions about Electron there are a lot of options. 
+
+Check out the [Electron community](https://www.electronjs.org/community#boilerplates). There are also a bunch of helpful people in this [Discord](https://discord.gg/electron) that should be willing to point you in the right direction. For your question, I'd recommend the Discord - we have many active help channels and mentors, as well as fellow devs, who can help you out.
```

Comment:
This already exists as https://raw.githubusercontent.com/electron/governance/main/playbooks/responses/not-an-issue.md but I like the text of this version better :+1:

However if this PR could remove not-an-issue.md to avoid duplication

Reply:
Oh I completely missed that 🤦‍♀️ Thanks! I'll just edit `not-an-issue` and remove this new one